src/RageSoundReader_ChannelSplit.cpp

```diff
@@ -196,7 +196,7 @@ int RageSoundSplitterImpl::ReadBuffer()
 	{
 		int iEraseFrames = iMinFrameRequested - m_iBufferPositionFrames;
 		iEraseFrames = std::min( iEraseFrames, (int) m_sBuffer.size() );
-		m_sBuffer.erase( m_sBuffer.begin(), m_sBuffer.begin() + iEraseFrames * m_pSource->GetNumChannels() );
+		m_sBuffer.erase( m_sBuffer.begin(), m_sBuffer.begin() + static_cast<std::size_t>(iEraseFrames) * m_pSource->GetNumChannels() );
```

Comment:
dont need std:: for well known int types 

https://google.github.io/styleguide/cppguide.html#Integer_Types

> Prefer to omit the std:: prefix for these types

Reply:
```suggestion
		m_sBuffer.erase( m_sBuffer.begin(), m_sBuffer.begin() + static_cast<size_t>(iEraseFrames) * m_pSource->GetNumChannels() );
```